### PR TITLE
LFH-59 queries to use titles rather than IDs to match API change]

### PR DESCRIPTION
**Pull request by vijamesDD**
Crafts, Category and Grade now to use titles rather than IDs

### CODE REVIEW

**Comment by vijamesDD** [comment]
Should wait until LFH-55 has been merged as I think some of the changes I've done in this PR are the same / overlap.

File: src/Skills/index.js [hunk 1]
```diff
@@ -66,8 +66,8 @@ export const Skills = ({ queryDetails }) => {
           }
         />
         <div>
-          <Chip label={skill.gradeId} className={classes.gradeChip} />
-          <Chip label={skill.categoryId} className={classes.categoryChip} />
+          <Chip label={skill.grade.title} className={classes.gradeChip} />
```
Comment (Caitlin-cooling) [inline src/Skills/index.js:69]: Think we might have to do some transformations on this. I tend to use lodash https://lodash.com/docs/4.17.15#startCase

File: README.md [hunk 2]
```diff
@@ -6,17 +6,26 @@
 
 
 
-## Available Scripts
+## Run locally
```
Comment (Caitlin-cooling) [inline README.md:9]: Thanks for updating this!

File: src/Skills/index.js (hunk 1 above)
Comment (Caitlin-cooling) [reply src/Skills/index.js:69]: Ah that's from LFH-55 🤦‍♀️ will change there

**Comment by Caitlin-cooling** [comment]
Should be good to rebase with master now :-)

**Comment by Caitlin-cooling** [comment]
Created a rebased branch with the changes here https://github.com/Caitlin-cooling/loofah-frontend/pull/25

**Comment by Caitlin-cooling** [comment]
Closing as merged rebased branch above

**Pull request closed without merging**